### DISCUSSION TITLE
Inquire about how to apply PDF internal search function.

### DISCUSSION THREAD

**Discussion by songminkyu** [cefsharp/CefSharp#3761] [Q&A]
Hello. I want to add PDF search functionality in CefSharp based on WPF. But it seems that there is no additional related code in xaml. If you have any other additional methods, please let me know.

**Comment by amaitland** [comment]
I don't believe text search in PDF is working currently 

https://magpcss.org/ceforum/viewtopic.php?f=6&t=17178

Hopefully this will be fixed as part of https://bugs.chromium.org/p/chromium/issues/detail?id=702993
